README.md

```diff
@@ -1,8 +1,76 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Nihar Sheth . Jarvis Consulting
 
+I am a computer engineering graduate from Ryerson University. I chose to pursue this field of study early on as I have always had an interest in computers and working with them. During my undergrad I gained experience and exposure to various fields such as software design, computer networks and security and data engineering. I found that data engineering captured my interest and I decided to further my career in it. What fascinates me about data engineering is its versatility as it applies to nearly every industry out there, and its importance is only growing with time.
+
+## Skills
+
+**Proficient:** Java, Python, C, Object-oriented design, Windows, Linux, Bash scripting, Git
+
+**Competent:** RDBMS/SQL, Agile/Scrum, Networks, GUI design, Numpy library, Pandas library, Seaborn library
+
+**Familiar:** Cybersecurity, Operating systems, Hardware engineering, MIPS assembly, VHDL
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_NiharSheth](https://github.com/jarviscanada/jarvis_data_eng_NiharSheth)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NiharSheth/tree/master/linux_sql)]: Implemented a monitoring agent that gathers Linux host information from various machines within a cluster and aggregates them in a single database using PostgreSQL within a Docker container. Static hardware information is stored for each host in one table while resource usage for each host is pushed to the database every minute automatically.
```

Comment:
Missing keywords.
![image](https://user-images.githubusercontent.com/58951265/136426888-97c15bf4-5f12-408f-852d-a43d7639205b.png)

Reply:
Please see the pforfile template file
https://github.com/jarviscanada/jarvis_profile_builder/blob/master/profile.yaml